plugins/speculation-rules/load.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Plugin Name: Speculation Rules
+ * Plugin URI: https://github.com/WordPress/performance/tree/trunk/modules/js-and-css/speculation-rules
+ * Description: Uses the Speculation Rules API to prerender linked URLs upon hover by default.
+ * Requires at least: 6.3
+ * Requires PHP: 7.0
+ * Version: 1.0.0
```

Comment:
After https://github.com/WordPress/performance/pull/993 this will need to be:

```suggestion
 * Version: 1.0.1
```